src/main/java/com/compilercharisma/chameleonbusinessstudio/entity/AppointmentEntity.java

```diff
@@ -20,108 +20,46 @@
+
     private String location = "online"; // default to online appointment
     
-    @Setter
-    @Getter
-    @Column(nullable=false)
+
     private String description = "";
     
-    @Setter
-    @Getter
-    @Column(nullable=false)
+
     private String restrictions = "";
     
-    @Setter
-    @Getter
-    @Column(nullable=false)
-    private boolean isCanceled = false;
-    
-    @Setter
-    @Getter
-    @Column(nullable=false)
-    private int totalSlots = 1; // default to one person per appointment
-    
-    @Setter
-    @Getter
-    @Column(nullable=false)
-    @ElementCollection
-    @CollectionTable(joinColumns = @JoinColumn(name = "appt_id"))
-    @JoinColumn(name = "appt_id")
-    @OnDelete(action = OnDeleteAction.CASCADE)
-    private Set<String> tags = new HashSet<>();
-    
-    @Setter
-    @Getter
-    @Column(nullable=false)
-    @ElementCollection
-    @CollectionTable(joinColumns = @JoinColumn(name = "appt_id"))
-    @JoinColumn(name = "appt_id")
-    @OnDelete(action = OnDeleteAction.CASCADE)
-    private Set<String> registeredUsers = new HashSet<>();
-    /*
-    registeredUsers column should have an FK relation to user email, but I don't
-    think it's possible to specify in JPA. Regardless, we'll be migrating to
-    Vendia in the near future, so it likely isn't worth investing time into.
-    */
+
+    private Boolean isCanceled = false;
     
+
+    private Integer totalSlots = 1; // default to one person per appointment
+
+    private List<String> participants;
     
-    @Override
-    public String toString(){
-        StringBuilder sb = new StringBuilder();
-        sb.append("Appointment:\n");
-        String[] lines = {
-            String.format("id : %d", id),
-            String.format("startTime : %s", startTime.toString()),
-            String.format("endTime : %s", endTime.toString()),
-            String.format("title : \"%s\"", title),
-            String.format("description : \"%s\"", description),
-            String.format("restrictions : \"%s\"", restrictions),
-            String.format("isCanceled : %b", isCanceled),
-            String.format("location: \"%s\"", location),
-            String.format("totalSlots: %d", totalSlots)
-        };
-        for(String line : lines){
-            sb.append(String.format("* %s%n", line));
-        }
-        
-        sb.append("* tags: \n");
-        tags.forEach((t)->{
-            sb.append(String.format("\t%s%n", t));
-        });
-        
-        sb.append("* registeredUsers: \n");
-        registeredUsers.forEach((email)->{
-            sb.append(String.format("\t%s%n", email));
-        });
-        
-        return sb.toString();
-    }
+//    @Setter
+//    @Getter
+//    @Column(nullable=false)
+//    @ElementCollection
+//    @CollectionTable(joinColumns = @JoinColumn(name = "appt_id"))
+//    @JoinColumn(name = "appt_id")
+//    @OnDelete(action = OnDeleteAction.CASCADE)
+//    private Set<String> tags = new HashSet<>();
```

Comment:
Please do not comment out features, as this leads to a cascading effect where we have to remember to re-enable what we've disabled.